cmd/create/cluster/cmd.go

```diff
@@ -136,37 +135,33 @@ func init() {
 		"",
 		"The Amazon Resource Name of the role that OpenShift Cluster Manager will assume to create the cluster.",
 	)
-	flags.MarkHidden("role-arn")
 	flags.StringVar(
 		&args.externalID,
 		"external-id",
 		"",
-		"A unique identifier that might be required when you assume a role in another account",
+		"A unique identifier that might be required when you assume a role in another account.",
 	)
-	flags.MarkHidden("external-id")
 	flags.StringArrayVar(
 		&args.operatorIAMRoles,
 		"operator-iam-roles",
 		nil,
-		"",
+		"List of name, namespace, and role ARNs used to perform credential "+
```

Comment:
I feel like we should mention its the OpenShift name and namespace to not confuse it with AWS?

Reply:
Fixed